ci/posix.yaml

```diff
@@ -33,6 +33,12 @@ jobs:
   - bash: conda env create --quiet --file=$(envFile) --name=dask-ml-test && conda list -n dask-ml-test
     displayName: "install"
 
+  - bash: |
+      conda install -y pytorch -c pytorch
+      pip install skorch
```

Comment:
This reinstalled a bunch of packages already present

```
Installing collected packages: scipy, tabulate, joblib, threadpoolctl, scikit-learn, skorch
Successfully installed joblib-0.16.0 scikit-learn-0.23.1 scipy-1.5.2 skorch-0.8.0 tabulate-0.8.7 threadpoolctl-2.1.0
```

Can you `pip install --no-deps skorch` and explicitly include any of skorch's other deps we don't already have?  Maybe just tabulate and tqdm (if they aren't already present).